### PR TITLE
Renamed all instance of Person to Patient

### PR DESCRIPTION
**Pull request by AaronJT1**
## What is changed
1.  For the source code, renamed all instances of 'Person' to 'Patient' in  all components (UI, Model, Storage,...) .
2. For the testing code, renamed all instances of 'Person' to 'Patient'.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -21,22 +21,19 @@
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.patient.*;
```
Comment (longnguyentan) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:24]: Using the '.*' form of import should be avoided

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -6,44 +6,41 @@
 
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.patient.*;
```
Comment (longnguyentan) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:9]: Using the '.*' form of import should be avoided

File: src/main/java/seedu/address/storage/JsonAdaptedPatient.java [hunk 3]
```diff
@@ -10,19 +10,16 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.patient.*;
```
Comment (longnguyentan) [inline src/main/java/seedu/address/storage/JsonAdaptedPatient.java:13]: Using the '.*' form of import should be avoided

**Review by longnguyentan — CHANGES_REQUESTED** [review]
@AaronJT1 @wujy28 @thienmy0 @RiyaMehta2211 Guys, remember to run `Checkstyle` before committing the code
@longnguyentan

**Comment by AaronJT1** [comment]
Hi, thanks for suggestions, have done the necessary checkstyles checks!

**Pull request merged**